crates/ssz_codegen/tests/expected_output/test_pragmas_inheritance.rs

```diff
@@ -161,6 +213,72 @@ pub mod tests {
                 pub b: Optional<u16>,
                 pub c: Optional<u8>,
             }
+            impl<H: tree_hash::TreeHashDigest> tree_hash::TreeHash<H> for Child {
+                fn tree_hash_type() -> tree_hash::TreeHashType {
+                    tree_hash::TreeHashType::StableContainer
+                }
+                fn tree_hash_packed_encoding(&self) -> tree_hash::PackedEncoding {
+                    unreachable!("StableContainer should never be packed")
+                }
+                fn tree_hash_packing_factor() -> usize {
+                    unreachable!("StableContainer should never be packed")
+                }
+                fn tree_hash_root(&self) -> H::Output {
+                    use tree_hash::TreeHash;
+                    use ssz_types::BitVector;
+                    let mut active_fields = BitVector::<5u64>::new();
+                    if self.a.is_some() {
+                        active_fields.set_bit(0usize);
+                    }
+                    if self.b.is_some() {
+                        active_fields.set_bit(1usize);
+                    }
+                    if self.c.is_some() {
+                        active_fields.set_bit(2usize);
+                    }
+                    let mut hasher = tree_hash::MerkleHasher::<H>::with_leaves(5usize);
+                    if let Some(ref a) = self.a {
+                        hasher
+                            .write(
+                                <_ as tree_hash::TreeHash<H>>::tree_hash_root(a).as_ref(),
+                            )
+                            .expect("tree hash derive should not apply too many leaves");
+                    } else {
+                        hasher
+                            .write(&[0u8; 32])
```

Comment:
All these `&[0u8; 32]` would break if we use any other hash that does not output a 32-byte slice.
Suggestion: `H::get_zero_hash_slice(0)`